controllers/metal3.io/baremetalhost_controller_test.go

```diff
@@ -1766,6 +1769,54 @@ func TestUpdateEventHandler(t *testing.T) {
 
 			expectedProcess: true,
 		},
+		{
+			name: "do-requeue-when-finalizers-are-different",
+			event: event.UpdateEvent{
+				ObjectOld: &metal3api.BareMetalHost{ObjectMeta: metav1.ObjectMeta{
+					Generation:  0,
+					Finalizers:  []string{},
+					Annotations: map[string]string{},
+				},
+					Status: metal3api.BareMetalHostStatus{
+						LastUpdated: &metav1.Time{Time: now.Time},
+					},
+				},
+				ObjectNew: &metal3api.BareMetalHost{ObjectMeta: metav1.ObjectMeta{
+					Generation:  0,
+					Finalizers:  []string{metal3api.BareMetalHostFinalizer},
+					Annotations: map[string]string{},
+				},
+					Status: metal3api.BareMetalHostStatus{
+						LastUpdated: &metav1.Time{Time: later},
+					}},
+			},
+
+			expectedProcess: true,
+		},
+		{
+			name: "do-not-requeue-when-status-lastupdated-are-different",
```

Comment:
Just my two cents, but given that there is already a test for [base case](https://github.com/metal3-io/baremetal-operator/pull/1881/files#diff-20c1a976622b8d54cf98349e3e6219778fa00e89ed669f1bb12cd778326469bbR1693), I would find a better name something like `do-not-requeue-when-only-status-lastupdated-are-different`

Reply:
I can update the name

Reply:
The name is updated.  I am resolving.